res/drawable/issue_event_closed.xml

```diff
@@ -0,0 +1,15 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillAlpha="1"
+        android:fillColor="#e65858"
```

Comment:
Yeah this doesn't work correctly on API < 21: 
![screenshot_20170217_122718](https://cloud.githubusercontent.com/assets/5156340/23063770/8829a136-f50c-11e6-874d-6981cf8fa9d5.png)

We should look into implementing support vector drawables in a separate pull request.